pypeit/core/save.py

```diff
@@ -399,47 +399,49 @@ def save_1d_spectra_fits(specobjs, header, outfile, helio_dict=None, telescope=N
     # Loop on detectors
     npix = 0
     ext = 0
-    for specobj in all_specobj:
-        if specobj is None:
+    for spec in specobjs:
```

Comment:
If ```specobjs``` is a ```SpecObjs``` object as the docs state, then I feel this should
be

```for spec in specobjs.specbojs```

I suspect the docs are wrong.

Reply:
Fixed.